src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -462,6 +474,11 @@ export default class RightPanel extends React.PureComponent<Props, State> {
                           disabled={!canEditItemMetadata}
                           onChange={this.handleChangeCategory}
                         />
+                        {!!data &&
```

Comment:
nit-pick, could we store this validation in a variable? 
something like:
```const showErrorMetrics = !!data && item.type === ItemType.WEARABLE && !isValidWearableMetrics(item.metrics, data.category as WearableCategory)```
so the JSX looks a bit cleaner, wdyt?

Reply:
sure! it will look cleaner 🙌